package.json

```diff
@@ -20,7 +20,7 @@
     "build": "babel ./modules -d lib --ignore '__tests__'",
     "build-umd": "NODE_ENV=production webpack modules/index.js umd/ReactRouter.js",
     "build-min": "NODE_ENV=production webpack -p modules/index.js umd/ReactRouter.min.js",
-    "lint": "eslint modules examples",
+    "lint": "eslint modules examples *.js",
```

Comment:
I'm intentionally only linting the `modules` and `examples` directories here. We don't need to lint everything.